### DISCUSSION TITLE
I'm newbie in discussion. Let's follow each other! 👋

### DISCUSSION THREAD

**Discussion by evan-hwang** [community/community#20188] [Other Feature Feedback, Questions, & Ideas]
Discussion! I didn'y know this function wa created!

Nice to mett you. I'm a Korean backend developer.
Let's follow each other and have fun interacting. 😀

Thx!

**Comment by diegoleondev** [comment]
Hi @evan-hwang 👋

**Reply by nao7711** [reply]
よろしくお願いします。😄

**Comment by mrtuvn** [comment]
just come along to say hi
Have a good day!

**Comment by melvincwng** [comment]
@evan-hwang , hello welcome to GitHub Community 👋

 안녕하세요 친구.

**Comment by uncle-bo** [comment]
hello~

**Comment by D-landJS** [comment]
I'am newbie too. I'm peruvian mobile developer, nice to meet u !
